### DISCUSSION TITLE
looking for an image for CubieAIO-A20

### DISCUSSION THREAD

**Discussion by frankipl** [johang/sd-card-images#47] [General]
Hi

I have tested bullseye images for Cubie based on A7 Cortex from this Site, but none works for me, just blank screen when I bool from the card
This one I need
http://www.cubietech.com/product-detail/cubieaio-a20/

I need this for Qt 5.8 with CAN modules in kernel, old Debian/Jessie does not work, I have updated my installation to Buster (with problems) but cant add CAN modules into kernel

Best Regards
Marek

**Comment by johang** [comment]
That board is not supported by mainline U-boot and kernel. I can only build mainline boards.

**Reply by frankipl** [reply]
OK. Thanks for info.
Cheers
Marek
